src/en/actions.md

```diff
@@ -13,141 +12,138 @@ against the schema defined in actions.yaml. See
 [Actions for the charm author](authors-charm-actions.html) for more
 information.
 
-Actions are sub-commands of the `juju action` command. To get more on their
-usage, use `juju action help`.
-
-Actions are exposed as of Juju 1.23. In Juju 1.22, they are only enabled in
-the CLI when `JUJU_DEV_FEATURE_FLAG=actions` is set.
-
-The following subcommands are specified:
+The following commands are specified for dealing with actions:
 
 ```no-highlight
-defined - show actions defined for a service
-do      - queue an action for execution
-fetch   - show results of an action by ID
-status  - show results of all actions filtered by optional ID prefix
+actions            - alias for 'list-actions'
+list-actions       - list actions defined for a service
+run-action         - queue an action for execution
+show-action-output - show results of an action by ID
+show-action-status - show results of all actions 
+                     filtered by optional ID prefix
```

Comment:
I know you were following what was there, but it makes more sense to me to do this by styling the commands as inline code and putting juju in front of them - e.g. 

`juju actions` - alias for list actions
`juju list actions` - list actions...